sparse/numba_backend/_common.py

```diff
@@ -2066,6 +2081,7 @@ def _support_numpy(func):
     we want to flag it and dispatch to NumPy.
     """
 
+    @wraps(func)
```

Comment:
This change makes the functions wrapped by `_support_numpy` show the right signature, and is an easy win.